generators/app/templates/src/containers/TextEditor/text-editor.style.js

```diff
@@ -0,0 +1,162 @@
+import styled from 'styled-components';
+
+import { media } from '../../utils';
```

Comment:
For consistency this should be imported from `@utils`

Reply:
ah ok